### DISCUSSION TITLE
please refund my copilot subscription because I can't make use of it despite my best efforts to do so

### DISCUSSION THREAD

**Discussion by sanikolov** [community/community#123046] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

This issue is a followup to ticket https://github.com/orgs/community/discussions/121386 which no one is monitoring any more.
Sequence of events
1. Paid for a copilot subscription 2 weeks ago - one payment for a year
2. Attempted to activate my copilot entitlement from Visual Studio 2022 (not Visual Studio Code which is different) - repeatedly received message "Sign-in is blocked".
3. Opened a ticket with github support regarding the sign-in issue. After multiple trials and errors there has been zero progress.

Again, I am requesting that your billing staff issue a refund.
Don't make me dispute the transaction with the credit card issuer.

**Comment by davecheney** [comment, marked as answer]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
